fe/src/main/java/org/apache/doris/alter/MaterializedViewHandler.java

```diff
@@ -260,7 +260,7 @@ public void processBatchAddRollup(List<AlterClause> alterClauses, Database db, O
 
                 // step 3 create rollup job
                 RollupJobV2 alterJobV2 = createMaterializedViewJob(rollupIndexName, baseIndexName, rollupSchema, addRollupClause.getProperties(),
-                        olapTable, db, baseIndexId);
+                        olapTable, db, baseIndexId, null);
```

Comment:
How about passing the olapTable's key type instead of null?

Reply:
OK